src/queues/job-queue.js

```diff
@@ -172,7 +172,7 @@ const initQueue = () => {
   /**
    * (START) General Queue
    */
-  BetterSocialQueue.setEventCallback(credderScoreQueue, credderScoreProcess);
+  // BetterSocialQueue.setEventCallback(credderScoreQueue, credderScoreProcess);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The line that sets the event callback for `credderScoreQueue` has been commented out. If this is intentional and you want to disable Credder functionality, it's fine. However, if you plan to re-enable this in the future, consider using a feature flag or configuration option instead of commenting out the code. This would make it easier to enable or disable the feature without modifying the codebase.

```diff
- // BetterSocialQueue.setEventCallback(credderScoreQueue, credderScoreProcess);
+ const enableCredder = false; // Or get this value from a config file or environment variable
+ if (enableCredder) {
+   BetterSocialQueue.setEventCallback(credderScoreQueue, credderScoreProcess);
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->